src/pages/RFOX/components/Shared/SharedMultiStepStatus.tsx

```diff
@@ -31,7 +31,6 @@ export type MultiStepStatusStep = {
   asset: Asset
   headerCopy: string
   serializedTxIndex: string | undefined
-  txHash: string | undefined
```

Comment:
There's just no point in passing both, deserializing the serialized Txid is 1 LoC